### PR TITLE
Added support in debian-like systems

### PR DESCRIPTION
**Pull request by adir-stratoscale**
@eliran-stratoscale @michaelk-stratoscale please review.

### CODE REVIEW

**Comment by eliran-stratoscale** [comment]
Validate that it still works for Centos

File: inaugurator/grub.py [hunk 1]
```diff
@@ -48,5 +48,13 @@ def setSerialDevices(serialDevices, destination):
 
 
 def install(targetDevice, destination):
-    chrootScript = 'grub2-install %s && grub2-mkconfig > /boot/grub2/grub.cfg' % targetDevice
-    sh.run("/usr/sbin/busybox chroot %s sh -c '%s'" % (destination, chrootScript))
+    try:
+        chrootScript = 'grub2-install %s && grub2-mkconfig > /boot/grub2/grub.cfg' % targetDevice
+        sh.run("/usr/sbin/busybox chroot %s sh -c '%s'" % (destination, chrootScript))
+        return '/boot/grub2/grub.cfg'
+    except:
```
Comment (eliran-stratoscale) [inline inaugurator/grub.py]: Print the exception.

Comment (adir-stratoscale) [inline inaugurator/grub.py]: done

**Comment by adir-stratoscale** [comment]
@eliran-stratoscale i tested this change in rackattack-virtual on both ubuntu 14.04 (trusty) and centos 7 (rootfs-star). i've managed to run playaround and connect to the allocated node via ssh.

**Comment by eliran-stratoscale** [comment]
Ship it!

**Comment by adir-stratoscale** [comment]
@michaelk-stratoscale please review.

**Pull request closed without merging**